wifistats/index.js

```diff
@@ -0,0 +1,129 @@
+const exec          = require('child_process').exec;
+const async         = require('async');
```

Comment:
Well the initial implementation was based on a similiar library, and we talked with callstats (where this will be used) to use callback, but later they moved to Promise.
I will try modifying it and make it promise based and will get rid of async.